### PR TITLE
hey shivam, I recommended good changes

### PR DESCRIPTION
**Pull request by shivamgoyal101**
added a line in git1 and deleted git9.

### CODE REVIEW

File: git1.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-First git push updated
\ No newline at end of file
+First git push updated
+
+change to learn git pull request.
```
Comment (shivamgoyal312) [inline git1.txt:3]: add 1 more line

Comment (shivamgoyal101) [reply git1.txt:3]: ok will do it

File: git9.txt [hunk 2]
```diff
@@ -1,2 +0,0 @@
-Hi this is from inside VS code.
```
Comment (shivamgoyal312) [inline git9.txt:1]: delete two more files

Comment (shivamgoyal312) [reply git9.txt:1]: yeah sure

Comment (shivamgoyal101) [reply git9.txt:1]: deleted as requested

File: git1.txt (hunk 1 above)
Comment (shivamgoyal101) [reply git1.txt]: added 1 more line

Comment (shivamgoyal312) [reply git1.txt]: nice

File: git9.txt (hunk 2 above)
Comment (shivamgoyal312) [reply git9.txt:1]: well done

**Pull request merged**